frontend/public/tutorial.md

```diff
@@ -43,11 +95,11 @@ const App = () => {
 };
 ```
 
-Now test out the "Add Todo" button!
+Now test out the **Add Todo** button! Notice as you add new to-dos, they appear in the view in real-time. To learn how to update a to-do, click **Next**.
 
-## Updating a Todo
+## Updating a To-do
 
-In `App.tsx`
+1. To update the contents of an existing to-do, we use the `update` method of the [Squid Client SDK](https://docs.squid.cloud/docs/development-tools/client-sdk/mutations#update). In `App.tsx`, update the `handleToggle` function with the following code:
```

Comment:
Unless I misunderstand the functionality, we aren't actually updating the content but rather updating its status. Not sure if content update is WIP though